Xamarin.PropertyEditing.Mac/Controls/FilePathEditorControl.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+using System.IO;
+using Xamarin.PropertyEditing.Common;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class FilePathEditorControl : PropertyEditorControl<PropertyViewModel<FilePath>>
```

Comment:
Most of this editor can be shared with a directory control, can we split it into a `abstract BasePathEditorControl` and a file implementation?

Reply:
done!